app/views/play/common/ChangeLanguageTab.js

```diff
@@ -182,48 +187,66 @@ module.exports = (ChangeLanguageTab = (function () {
 
     buildCodeFormats () {
       const $select = this.$el.find('#option-code-format')
-      if (!utils.isMobile()) {
-        $select.fancySelect()
+      if (!utils.isMobile() && $select.length) {
+        if (this.codeformatChoices) {
+          this.codeformatChoices.destroy()
+        }
+        this.codeformatChoices = new Choices($select[0], {
+          searchEnabled: false,
+          itemSelectText: '',
+          classNames: {
+            containerInner: ['choices__inner', 'trigger'],
+            listDropdown: ['options'],
+            itemChoice: ['item'],
+            selectedState: ['selected'],
+            highlightedState: ['hover'],
+          },
+        })
       }
-      $select.parent().find('.options li').each(function () {
+      $select.parent().parent().find('.options .item').each(function () {
         const formatName = $(this).text()
         const formatID = $(this).data('value')
         const blurb = $.i18n.t(`choose_hero.${formatID}_blurb`.replace(/-/g, '_'))
         if (formatName.indexOf(blurb) === -1) { // Avoid doubling blurb if this is called 2x
           return $(this).text(`${formatName} - ${blurb}`)
         }
       })
-      if ($select.parent().find('.options li').length === 1) {
-        $select.trigger('disable.fs')
-      } else {
-        $select.trigger('enable.fs')
-      }
     }
 
     buildCodeLanguages () {
       const $select = this.$el.find('#option-code-language')
-      if (!utils.isMobile()) {
-        $select.fancySelect()
+      if (!utils.isMobile() && $select.length) {
+        if (this.languageChoices) {
+          this.languageChoices.destroy()
+        }
+        this.languageChoices = new Choices($select[0], {
```

Comment:
language and codeformat use the same object as second parameter, DRY code?

Reply:
ok. yes we only use fancy-select here lol